opensrs/constants.py

```diff
@@ -1 +1,12 @@
 AUTO_RENEWED_TLDS = ('de', 'dk', 'za', 'at', 'fr')
+
+
+class OrderProcessingMethod(object):
```

Comment:
What do you think about `...Methods`?  (since it's a container for collection of order processing methods/options)